build.psm1

```diff
@@ -220,6 +220,10 @@ function Start-PSBuild {
         [switch]$NoPSModuleRestore,
```

Comment:
@rjmholt, your last commit had 1 failures in `PowerShell-CI-windows`
Invoke-WebRequest tests.Denial of service.Image Parsing
```powershell
Expected the actual value to be greater than 10, but got 6.95489657038942.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\WebCmdlets.Tests.ps1: line 1897
1897:             $pathologicalRatio | Should -BeGreaterThan 10
```